org.eclipse.scanning.server/src/org/eclipse/scanning/server/servlet/AcquireRequestHandler.java

```diff
@@ -67,40 +69,79 @@ public AcquireRequest process(AcquireRequest bean) throws EventException {
 			bean.setStatus(Status.COMPLETE);
 			bean.setMessage(null);
 		} catch (Exception e) {
-			e.printStackTrace();
 			bean.setStatus(Status.FAILED);
 			bean.setMessage(e.getMessage());
 			logger.error("Cannot acquire data for detector " + getBean().getDetectorName(), e);
 			throw (e instanceof EventException) ? (EventException)e : new EventException(e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the usage of the "instanceof" operator by a catch block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1193)